catalyst_voices/packages/internal/catalyst_voices_view_models/lib/src/proposal/guidance/guidance.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:catalyst_voices_view_models/src/proposal/guidance/guidance_type.dart';
+import 'package:equatable/equatable.dart';
+
+final class Guidance extends Equatable {
+  final String title;
+  final String description;
```

Comment:
this `title` and `description` are not easy to localised now.

One potential solution is to make this class abstract and then create implementations with correct localisations.

```dart
abstract base class Guidance ...

String localizedTitle(BuildContext context);

String localizedDescription(BuildContext context);
```

example:

```dart
abstract interface class SectionStep implements SectionsListViewItem {
  int get id;

  int get sectionId;

  SectionStepId get sectionStepId;

  bool get isEnabled;

  bool get isEditable;

  String localizedName(BuildContext context);
}
```

```dart
final class ProposalSolution extends WorkspaceSection {
  const ProposalSolution({
    required super.id,
    required super.steps,
  });

  @override
  String localizedName(BuildContext context) {
    return 'Proposal solution';
  }
}
```

Reply:
I don't believe this case applies to this class. This data should come from the backend. We will not have a translation for every possible title and description of the guidance. Guidance will be created by the admin in the campaign builder.